README.md

```diff
@@ -47,6 +47,7 @@ See public/licenses.txt
 * A Ruby manager such as [RVM](http://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv) is strongly recommended.
 * (For shapefile support) access to an [ogre](http://ogre.adc4gis.com) server
 * (For placename completion) a [GeoNames](http://www.geonames.org) account
+* For automatic spatial and temporal extraction from the search text, clone and set up an [EDSC-NLP](https://github.com/mightynimble/edsc-nlp) server
```

Comment:
Should this live under NASA instead of your personal GitHub? I'm not sure what we need to do for that